models/config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	port: 3000,
```

Comment:
What's this ?

Reply:
port of the server

Reply:
Why is it here, is the meaning of my question

Reply:
Hard coding information, so i put any hard coding info in config.js.

Reply:
An app port doesn't belong in a config file within `models`, that's what I'm saying. Not the correct place for it.